test/fixtures/yaml/invalid.yaml

```diff
@@ -0,0 +1,17 @@
+object:
```

Comment:
We mostly use eslint-disable-next-line when possible. Is that possible here?

Reply:
Done